deploy/ansible/worker/include/setup_docker.yml

```diff
@@ -39,25 +39,6 @@
         name: nucypher
         group: docker
 
-    - name: Download docker-compose
```

Comment:
This code is in the worker directory, is it also unused?

Reply:
I think (?) this code was added in order to have a common code/location to obtain `docker-compose`. Once obtained it can then be used with the monitor - I'm not sure that docker-compose was utilized anywhere else - @vepkenez ?

Basically this removal matches the additions made here - https://github.com/nucypher/nucypher/pull/2801/commits/cb8f40e6bc4ea19fe26f9f21d19fda042df23544#diff-43c64472a931bf0fb6bc4f21d485f2e3584c45ccb554dfda94fd17153f9c69c9R42 - unless it is still useful.

Reply:
As noted below, I am planning on potentially using docker-compose for future cloudworkers.  It is useful to keep.